zebra-rpc/src/methods/get_block_template_rpcs/types/get_block_template.rs

```diff
@@ -1,57 +1,96 @@
 //! The `GetBlockTempate` type is the output of the `getblocktemplate` RPC method.
 
-use crate::methods::get_block_template_rpcs::types::{
-    coinbase::Coinbase, default_roots::DefaultRoots, transaction::Transaction,
+use zebra_chain::{amount, block::ChainHistoryBlockTxAuthCommitmentHash};
+
+use crate::methods::{
+    get_block_template_rpcs::types::{
+        default_roots::DefaultRoots, transaction::TransactionTemplate,
+    },
+    GetBlockHash,
 };
 
 /// Documentation to be added after we document all the individual fields.
 #[derive(Clone, Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
 pub struct GetBlockTemplate {
```

Comment:
These fields now have the correct types, but the data will be incomplete until we finish the RPC.